tests/mock.c

```diff
@@ -306,7 +306,9 @@ int ioctl(int fd, unsigned long request, ...)
 			if (gEnableIRQ && fme_irq->evtfd >= 0) {
 				uint64_t data = 1;
 				// Write to the eventfd to signal one IRQ event.
-				write(fme_irq->evtfd, &data, sizeof(data));
+				if (write(fme_irq->evtfd, &data, sizeof(data)) != sizeof(data)) {
+					FPGA_ERR("IRQ write < 8 bytes");
```

Comment:
`!=` is no the same as `<`

Reply:
True, but it would't be greater and it's not equal, so it must be less.. :)